web/message/Message.jsx

```diff
@@ -34,9 +35,23 @@ class Message extends Component {
   }
 
   renderHTMLMessage() {
-    let classes = 'message-body m-t-midxs';
-    const message = this.props.message.get('messageHtml').replace(/@([A-Za-z0-9_-]+)/g, this.renderFormattedMention);
-    return <div className={classes} dangerouslySetInnerHTML={ { __html: message } }></div>;
+    let classes = 'message-body m-t-midxs pos-rlt visible-on-hover-trigger';
```

Comment:
To clean this up to bit I would create three more render functions:

- `renderDeletedMessage`
- `renderMessageForMod`
- `renderMessage`

And have this method function only as a switch to call the correct render function.